README.rst

```diff
@@ -78,8 +78,8 @@ Same as usual, it can be installed with `pip`
 
 .. code-block:: bash
```

Comment:
It would be better to change `code-block:: console` to show better syntax highlight for interactive session example